src/components/home/components/volt_sale_card.jsx

```diff
@@ -23,6 +23,8 @@ const VoltSaleCard = () => {
 
   const [tokenSaleAddress, setTokenSaleAddress] = useState(null)
 
+  const [tokenPriceSelected, setTokenPriceSelected] = useState('')
```

Comment:
@RudeWolfCoding Is it really necessary to separate the to `tokenSaleAddress` & `tokenPriceSelected`?

Reply:
necessary for the volt sale card to show the appropriate calculations of Totals to the user. tokenPriceSelected is getting information from the price select input.